application/testing/CMakeLists.txt

```diff
@@ -721,57 +721,61 @@ if(F3D_PLUGIN_BUILD_DRACO)
 
 endif()
 
-if(F3D_PLUGIN_BUILD_EXODUS)
-  f3d_test(NAME TestExodus DATA disk_out_ref.ex2 ARGS --load-plugins=exodus -s --camera-position=-11,-2,-49)
+if(F3D_PLUGIN_BUILD_HDF)
+  f3d_test(NAME TestVTKHDF DATA blob.vtkhdf ARGS --load-plugins=hdf -s)
```

Comment:
you need to put this test behind a VTK_VERSION check 9.3 or 9.4 I think. Better if you know the exact date.

Reply:
waiting for CI to tell me if 9.3 is fine